src/components/portrait/portrait-initials.component.js

```diff
@@ -100,6 +111,7 @@ class PortraitInitials extends React.Component {
         shape={shape}
         theme={theme}
         {...otherProps}
+        {...this.marginProps()}
```

Comment:
same as above

Reply:
Removed as requested. Thanks for spotting this 👍🏻